FunctionalTableData/Separator.swift

```diff
@@ -21,11 +21,12 @@ public class Separator: UIView {
 	
 	/// The style for table cells used as separators.
 	///
-	/// The options are `full`, `inset`, and `moreInset`
-	public enum Style {
+	/// The options are `full`, `inset`, `moreInset`, `custom`
+	public enum Style: Equatable {
 		case full
 		case inset
 		case moreInset
+		case custom(leftInset: CGFloat, rightInset: CGFloat, layoutMarginsRelative: Bool)
```

Comment:
nit: maybe could name these `leadingInset` and `trailingInset` to match the constraints naming below?

```suggestion
		case custom(leadingInset: CGFloat, trailingInset: CGFloat, layoutMarginsRelative: Bool)
```